source/qb64pe.bas

```diff
@@ -13334,9 +13319,6 @@ FUNCTION ParseCMDLineArgs$ ()
                 token$ = MID$(token$, 3)
 
                 SELECT CASE LCASE$(LEFT$(token$, INSTR(token$, "=") - 1))
-                    CASE ":useminiaudio"
```

Comment:
This is an interesting case, I'd be tempted to still allow `useminiaudio=true` from a backwards compatibility perspective. But realistically nobody is using this so let's just take it out :D

Reply:
BTW, thanks. Your documentation (QB64pe/docs) made this whole thing a breeze for me. I picked up a couple of useful things from there. Top stuff!